integration-cli/docker_cli_run_unix_test.go

```diff
@@ -134,25 +134,24 @@ func (s *DockerSuite) TestRunAttachDetach(c *check.C) {
 	}
 }
 
-// "test" should be printed
-func (s *DockerSuite) TestRunEchoStdoutWithCPUQuota(c *check.C) {
+func (s *DockerSuite) TestRunWithCPUQuota(c *check.C) {
 	testRequires(c, cpuCfsQuota)
 
-	out, _, err := dockerCmdWithError("run", "--cpu-quota", "8000", "--name", "test", "busybox", "echo", "test")
-	c.Assert(err, checker.IsNil, check.Commentf("failed to run container, output: %q", out))
-	out = strings.TrimSpace(out)
-	c.Assert(out, checker.Equals, "test", check.Commentf("container should've printed 'test'"))
+	file := "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"
+	out, _ := dockerCmd(c, "run", "--cpu-quota", "8000", "--name", "test", "busybox", "cat", file)
+	c.Assert(out, checker.Contains, "8000")
```

Comment:
In case `-m`, I got some warnings about memory swap is not supported in `out`, which is intended, so I use `Contains` instead, maybe we can use `Equals` for other cases that won't cause any warnings because of the different kernel support.